src/fu-util.c

```diff
@@ -4018,6 +4048,89 @@ fu_util_get_bios_setting(FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_emulation_tag(FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(FwupdDevice) dev = NULL;
+	g_autofree gchar *cmd = g_strdup_printf("%s emulation-save", g_get_prgname());
+
+	/* set the flag */
+	dev = fu_util_get_device_or_prompt(priv, values, error);
+	if (dev == NULL)
+		return FALSE;
+	if (!fwupd_client_modify_device(priv->client,
+					fwupd_device_get_id(dev),
+					"Flags",
+					"emulation-tag",
+					priv->cancellable,
+					error))
+		return FALSE;
+
+	/* TRANSLATORS: these are instructions on how to generate the data, and the %1 is a command
+	 * like 'fwupdmgr emulation-save' */
+	g_print(_("Now unplug and replug the device, do the update, and then call %s"), cmd);
```

Comment:
I think this works with upgrade/downgrade/reinstall whatever.  So perhaps tell them to "install the firmware"

Reply:
Fixed, thanks.